mappings/net/minecraft/client/render/BufferRenderer.mapping

```diff
@@ -3,8 +3,8 @@ CLASS net/minecraft/class_286 net/minecraft/client/render/BufferRenderer
 	COMMENT BufferBuilder}.
 	FIELD field_38982 currentVertexBuffer Lnet/minecraft/class_291;
 	METHOD method_34420 reset ()V
-	METHOD method_43433 drawWithShader (Lnet/minecraft/class_287$class_7433;)V
-		COMMENT Draws {@code buffer} using the shader specified with {@link
+	METHOD method_43433 drawWithGlobalProgram (Lnet/minecraft/class_287$class_7433;)V
```

Comment:
@enbrain I wonder if the former name is better, "shader" makes more sense in this context. Should we revert this?

Reply:
Can you elaborate why?

Reply:
"Shader" is less ambiguous than "program". After all, a program is a set of shaders.

Reply:
Imo, it's expected knowledge that "program" means a shader program in this context. If ambiguity is really an issue, `drawWithGlobalShaderProgram` would be better. The point of this refactor is that a mere "shader" should never be used to refer to a shader program.

On a side note, almost all content mods shouldn't call this method directly anyway. There are higher level API `Tessellator.draw` and `RenderLayer.draw`.